### DISCUSSION TITLE
CATS CATS CATS!

### DISCUSSION THREAD

**Discussion by itserindean** [hackforla/food-oasis#1523] [Polls]
lets get the purrty started

**Comment by staceyrebekahscott** [comment]
@itserindean A tabby cat is defined by its fur pattern: 

A tabby is any domestic cat with a distinctive 'M'-shaped marking on its forehead; stripes by its eyes and across its cheeks, along its back, and around its legs and tail; and, characteristic striped, dotted, lined, flecked, banded, or swirled patterns on the body—neck, shoulders, sides, flanks, chest, and abdomen. (from Wikipedia) 

Therefore it is possible for a cat to be both a tabby cat and an orange or gray cat. My cat Uno is an orange tabby cat. I don't think this warrants another poll, but thought it was worth clarifying since there was some questions about this during the meeting ;-)
